locales/pt_BR/json.json

```diff
@@ -906,7 +906,7 @@
     "This subscription cannot be resumed. Please create a new subscription.": "Esta assinatura não pode ser retomada. Por favor, crie uma nova assinatura.",
     "This subscription has expired and cannot be resumed. Please create a new subscription.": "Esta assinatura expirou e não pode ser retomada. Por favor, crie uma nova assinatura.",
     "This team has too many team members for the selected plan.": "This team has too many team members for the selected plan.",
-    "This team is not eligible for this plan.": "This team is not eligible for this plan.",
+    "This team is not eligible for this plan.": "Esta equipa não é elegível para este plano.",
```

Comment:
The translation of team into Brazilian Portuguese is incorrect. We can use `Equipe` or `Time`.